.github/workflows/deploy_and_test.yml

```diff
@@ -0,0 +1,61 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+    paths-ignore:
+      - "*.md"
+      - "*.png"
+      - "*.gif"
+      - "*.svg"
+      - "docs/**"
+      - ".vscode/**"
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+      - "*.md"
+      - "*.png"
```

Comment:
Just for my understanding - could it be a problem if you have test data as a `.png` which you change in the PR because the tests would not be triggered?

Reply:
yes, that would be a problem